src/sage/rings/polynomial/multi_polynomial_libsingular.pyx

```diff
@@ -2077,30 +2077,30 @@ cdef class MPolynomial_libsingular(MPolynomial):
             raise TypeError("number of arguments does not match number of variables in parent")
 
         try:
-            # Attempt evaluation via singular.
             coerced_x = [parent.coerce(e) for e in x]
         except TypeError:
-            # give up, evaluate functional
+            # give up, evaluate symbolically
             y = parent.base_ring().zero()
-            for (m,c) in self.dict().iteritems():
-                y += c*mul([ x[i]**m[i] for i in m.nonzero_positions()])
+            for m, c in self.dict().iteritems():
+                y += c * mul([x[i]**m[i] for i in m.nonzero_positions()])
             return y
 
-        cdef poly *res    # ownership will be transferred to us in the next line
-        singular_polynomial_call(&res, self._poly, _ring, coerced_x, MPolynomial_libsingular_get_element)
```

Comment:
I think the comments regarding object ownership may be valuable and should be kept

Reply:
Okay.